Diagnostics/FreeBusyChecker/Functions/hostOutput.ps1

```diff
@@ -0,0 +1,131 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', '', Justification = 'Variables are being used in functions')]
+param (
+    [Parameter(Mandatory = $false)]
+    $TestingDAuthConfiguration = "---------------------------------------Testing DAuth Configuration-----------------------------------------------",
+    $TestingOAuthConfiguration = " ---------------------------------------Testing OAuth Configuration----------------------------------------------- ",
+    $CollectingExoAvailabilityInformation = " Collecting Exchange Online Availability Information",
+    $ExchangeOnlinePowershellModuleMessage = "`n Exchange Online Powershell Module is required to Check Free Busy Configuration on Exchange Online side. Installing Module",
+    $ExchangeOnlineModuleAvailableMessage = "`n ExchangeOnlineManagement module is available.",
+    $TestingExoDAuthConfiguration = " ---------------------------------------Testing DAuth Configuration----------------------------------------------- ",
+    $TestingExoOAuthConfiguration = " ---------------------------------------Testing OAuth Configuration----------------------------------------------- ",
+    $ThatIsAllForTheExchangeOnlineSide = " That is all for the Exchange Online Side"
+)

```

Comment:
This is all not needed.

Reply:
Removed from code. Correction will be available when changes are pushed.